src/test/ui/parser/issue-87812-path.rs

```diff
@@ -0,0 +1,13 @@
+#![deny(break_with_label_and_loop)]
+
```

Comment:
Why is this lint denied in this test?

Reply:
This is the lint that gets triggered. It’s `warn` by default.

Reply:
But I don't see the lint being emitted in the stderr...?

Reply:
@camelid You're right. I had this test mixed up with the other one.

It's fixed now.